.azure/templates/jobs/build_java.yaml

```diff
@@ -31,7 +30,7 @@ jobs:
           BRANCH: $(Build.SourceBranch)
           TESTCONTAINERS_RYUK_DISABLED: "TRUE"
           TESTCONTAINERS_CHECKS_DISABLE: "TRUE"
-          MVN_ARGS: "-e -V -B"          
+          MVN_ARGS: "-e -V -B -Dfailsafe.rerunFailingTestsCount=2"
```

Comment:
why is it needed now and it wasn't before?

Reply:
Oh, that's leftover. I wanted to double-check if the issue persists even after a re-try. I will remove it